auth_session_timeout/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# (c) 2015 ACSONE SA/NV, Dhinesh D
+# (c) 2015 ACSONE SA/NV, Dhinesh D, Jesse Morgan
```

Comment:
Thanks for the heads-up. 
I did in fact only change the copyright headers on the files I modified to begin with, but when I was 'polishing it up' to submit as a pull request I went back and added to the other files.

I guess the thought was that the enter module would be subject to copyright rather than each file within it.

No matter, I will change it :)